### DISCUSSION TITLE
Is it possible to remove kube-rbac-proxy

### DISCUSSION THREAD

**Discussion by zerunhu** [kubernetes-sigs/kubebuilder#3337] [Support] [answered]
I checked the information about kube-rbac-proxy and I think it is adding permissions to the controller so that the controller can access the Kubernetes API
But I see that kubebuilder minifest(./config/manager/manager.yaml) also has serviceAccountName field, I think it can achieve the same effect as kube-rbac-proxy, because I often need to deploy controllers privately, so pulling kube-rbac-proxy image will increase the complexity of my work, so can I remove the kube-rbac-proxy related information and just use serviceAccount.
I've already removed and tested the controller functionality without problems, I don't know if there will be hidden bugs

**Comment by camilamacedo86** [comment, marked as answer]
Yes, you can remove it. Yu need to remove from the scaffold all related files. See the auth_proxy ones inside of config/ dir.

**Comment by camilamacedo86** [comment]
Closing since it is answered.
